src/Sulu/Bundle/TagBundle/Tests/Unit/Search/TagsConverterTest.php

```diff
@@ -23,16 +23,6 @@ class TagsConverterTest extends TestCase
 {
     use ProphecyTrait;
 
-    public function testConvert(): void
-    {
-        $tagManager = $this->prophesize(TagManager::class);
-        $tagsConverter = new TagsConverter($tagManager->reveal());
-
-        $tagManager->resolveTagNames(['Tag1', 'Tag2', 'Tag3'])->willReturn([1, 2, 3]);
-
-        $this->assertEquals([1, 2, 3], $tagsConverter->convert(['Tag1', 'Tag2', 'Tag3']));
-    }
-
```

Comment:
This behaviour has been removed. Since we always have to pass 2 parameters to the function.